### ISSUE TITLE
Using UniTS on Unlabeled Data for Anomaly Detection Task

### ISSUE THREAD

**Issue by thisthq** [mims-harvard/UniTS#16]
Hi,
Thank you for your contributions.
I am currently working on an anomaly detection task on a multivariate time series dataset, but the issue is that this dataset currently does not have labels. I want to know if UniTS can be directly applied to this dataset, and if I can extract a portion of the dataset to fine-tune it.

Thank you!

**Comment by gasvn** [comment]
We only do the few-shot finetuning on the anomaly detection task. If you want to use UniTS for labeling the new dataset, I suggest you collect are the anomaly detection datasets you have and finetune a new UniTS to do so.

**Comment by thisthq** [comment]
> We only do the few-shot finetuning on the anomaly detection task. If you want to use UniTS for labeling the new dataset, I suggest you collect are the anomaly detection datasets you have and finetune a new UniTS to do so.

Yes, I intend to use UniTS to label my unlabeled dataset. Would it be appropriate if I loaded your pre-trained UniTS model (units_x32_pretrain_checkpoint.pth) and tested it on my unlabeled dataset without evaluating the results obtained from the model, since there are no ground truth labels available?

Regarding your advice, I apologize, but I am still unsure. Could you please provide me with further guidance?

Thank you.
